src/Validation/Validation.php

```diff
@@ -216,6 +216,36 @@ public static function cc($check, $type = 'fast', $deep = false, $regex = null)
     }
 
     /**
+     * Used to check the count of a given value of type string, int, or array.
+     *
+     * If a string value is passed the string length is used as count.
+     *
+     * @param array|int|string $check1 The value to check the count on.
+     * @param string $operator Can be either a word or operand
+     *    is greater >, is less <, greater or equal >=
+     *    less or equal <=, is less <, equal to ==, not equal !=
+     * @param int $expectedCount The expected count value.
+     * @return bool Success
+     */
+    public static function count($check1, $operator, $expectedCount)
+    {
+        $count = null;
+        if (is_array($check1) || $check1 instanceof \Countable) {
+            $count = count($check1);
+        } elseif (is_string($check1)) {
+            $count = mb_strlen($check1);
+        } elseif (is_int($check1)) {
```

Comment:
I strongly disagree with that but fine, I've removed it. It was just convenient.

There is no equals, just equalTo and it does not allow me to take a string and compare the count of it to something. http://api.cakephp.org/3.2/source-class-Cake.Validation.Validation.html#592 In fact it's just a strong typed comparison.

lengthBetween requirs a min and max value. http://api.cakephp.org/3.2/source-class-Cake.Validation.Validation.html#121

Both don't do the same my implementation allowed you to do.